rbac/rbac.go

```diff
@@ -49,10 +49,22 @@ type RoleBinding struct {
 	Roles    []string  `json:"roles"`
 }
 
+// ExtraAttributes contains extra data about the request that can be used to make a more precise authorization decision.
+type ExtraAttributes struct {
+	Logs *LogsExtraAttributes `json:"logs,omitempty"`
+}
+
+// LogsExtraAttributes contains extra attributes about a logs request.
+type LogsExtraAttributes struct {
+	Selectors         map[string][]string `json:"selectors,omitempty"`
+	WildcardSelectors bool                `json:"wildcardSelectors,omitempty"`
+	MetadataOnly      bool                `json:"metadataOnly,omitempty"`
+}
```

Comment:
Similar as mentioned in the opa-openshift repo: https://github.com/observatorium/opa-openshift/pull/20#discussion_r1322528010